cmd/agent/dist/conf.d/containerd.d/conf.yaml.default

```diff
@@ -0,0 +1,18 @@
+init_configs:
+instances:
+  -
+    # list filters of events you which to collect
+    # https://github.com/containerd/containerd/blob/master/runtime/events.go#L19-L42
+    # The check only supports tasks, images and containers, see pkg/collector/corechecks/containers/containerd_events.go.
+    filters:
+      - "/containers/create"
+      - "/containers/delete"
+      - "/containers/update"
+      - "/images/update"
+      - "/images/create"
+      - "/images/delete"
+      - "/tasks/oom"
+      - "/tasks/delete"
+    # Default to k8s.io per https://github.com/containerd/cri/blob/release/1.2/pkg/constants/constants.go#L22-L23
+    # Since this check is only supported in Kubernetes
+    namespace: "k8s.io"
```

Comment:
Is this option used?

Reply:
Ah, I did not update the yaml - Indeed it's not used anymore as it's a integration wide config, not specific to the check